core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/obtainTheOSName/ObtainTheOSNameUnitTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.obtainTheOSName
```

Comment:
We follow these guidelines for package names:
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

(all lowercase)
```suggestion
package com.baeldung.obtaintheosname
```

Reply:
@sk1418 this one is not showing up in the PR

Reply:
using camelcase as package names are defined in Kotlin's official naming convention. Also it follows in baeldung's Kotlin naming convention (I suggested it =) )
Link: https://docs.google.com/document/d/1I6eu-eH5D1E0Mkl0Al-4xTgYiK2Ny0PPGtXaN2ToepI/edit
But I changed the package to `obtainTheOsName`.

Reply:
> We follow these guidelines for package names: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names
> 
> (all lowercase)

Thanks. the link is for Java code style instead of Kotlin. Kotlin's convention allows camelcase package names.
https://kotlinlang.org/docs/coding-conventions.html#naming-rules

Reply:
Got it, last time I knew we were following Java conventions for Kotlin, but looks like that has been changed since I last looked, thanks for letting me know!